build/bubbleWrapper.ts

```diff
@@ -118,15 +118,36 @@ export class BubbleWrapper {
             password: signingInfo.storePassword,
             keypassword: signingInfo.keyPassword,
             alias: signingInfo.alias,
-            fullName: signingInfo.fullName,
-            organization: signingInfo.organization,
-            organizationalUnit: signingInfo.organizationalUnit,
-            country: signingInfo.countryCode
+            fullName: this.getEscapedSigningKeyText(signingInfo.fullName),
+            organization: this.getEscapedSigningKeyText(signingInfo.organization),
+            organizationalUnit: this.getEscapedSigningKeyText(signingInfo.organizationalUnit),
+            country: this.getEscapedSigningKeyText(signingInfo.countryCode)
         };
         
         await keyTool.createSigningKey(keyOptions, overwriteExisting);
     }
 
+    private getEscapedSigningKeyText(input: string): string {
```

Comment:
This could probably be in a utils file or similar, but thats a tiny thing that could be cleaned up in a seperate PR

Reply:
Yeah, agreed. Initially, I had this code in the front-end...but then I realized that users don't really care about **how** their text is sent to the Android key signing util - so I just escaped problematic text right before sending it into the util.

I agree though that it could be better placed.

Reply:
FWIW, latest Bubblewrap core should be escaping those when generating the key: https://github.com/GoogleChromeLabs/bubblewrap/blob/df3443a0269b280a173df2ecb7a379faf296babb/packages/core/src/lib/jdk/KeyTool.ts#L59-L89